library/headers.inc.php

```diff
@@ -37,4 +37,5 @@ function include_js_library($path)
 ?>
 <script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/textformat.js"></script>
```

Comment:
Or we can go with above function suggested by @aethelwulffe and then have proper documentation for the arguments , similar to the one we have for datatypes. I'm fine with both.